package.json

```diff
@@ -65,25 +65,25 @@
     ]
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.15.8",
     "@ember-data/rfc395-data": "^0.0.4",
     "@glimmer/syntax": "^0.84.2",
+    "@typescript-eslint/parser": "^5.31.0",
+    "@typescript-eslint/scope-manager": "^6.2.0",
     "css-tree": "^2.0.4",
     "ember-rfc176-data": "^0.3.15",
-    "ember-template-imports": "^3.4.2",
-    "ember-template-recast": "^6.1.4",
+    "ember-template-tag": "^2.2.3",
```

Comment:
any reason to not use `content-tag`?

Reply:
content-tag only transforms the content. it does not provide any info about the changes done.
It also does not guarantee the same code syntax & spacing, as its not a recast and also adds import statement at the top, which changes all lines.
i tried also to use source mapping, but it doesn't matter if the syntax is changed.